controllers/members.js

```diff
@@ -46,7 +49,33 @@ const getIdleMembers = async (req, res) => {
   }
 }
 
+/**
+ * Makes a new member a member
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+const moveToMembers = async (req, res) => {
+  try {
+    const { username } = req.params
+    const result = await fetchUser({ username })
+    if (result.userExists) {
+      const alreadyMember = await memberQuery.moveToMembers(result.user.id)
+      if (alreadyMember) {
+        return res.status(400).json({ message: 'User Already is a member' })
+      }
```

Comment:
We can use boom 400 here:
https://www.npmjs.com/package/boom#boombadrequestmessage-data